### ISSUE TITLE
using GA with LSTM model

### ISSUE THREAD

**Issue by teodorkasap** [run-ai/runai#15]
I am trying to use the GA wrapper with an LSTM model but get this error:

Cannot reconcile tf.cond 0-th outputs:
  branches returned: (<tensorflow.python.framework.indexed_slices.IndexedSlices object at 0x7f60740c0610>, <tf.Tensor 'training/Adam/runai/add:0' shape=(100000, 256) dtype=float32>)

my model looks like:
model = Sequential()
model.add(Embedding(vocabulary_size, 256, input_length=X.shape[1]))
model.add(Dropout(0.3))
model.add(Bidirectional(LSTM(256, return_sequences=True)))
model.add(Dropout(0.5))
# model.add(Bidirectional(CuDNNGRU(256, return_sequences=True)))
model.add(AttentionLayer(name='attention',activation='sigmoid'))
model.add(BatchNormalization())
model.add(Dense(2, activation='sigmoid',kernel_regularizer=regularizers.l2(0.01),
                activity_regularizer=regularizers.l1(0.01)))

**Comment by levosos** [comment]
Hi @teodorkasap and thank you for reaching out!

Indeed we haven't tested GA on an LSTM model, and this is also the reason why there isn't any example of such a model.

If you are able to share a fully working, self-contained, minimal version of your model (using one of the official datasets) we would be happy to test it ourselves and come back with a solution.

If you are not able to share this with us and you are familiar with an open-source LSTM model that is similar to yours, please provide a reference to it.

In case we will not be able to reproduce it ourselves, please provide as much information as you can and we'll try to figure that out.

Thank you in advance! and thanks again for reaching out!

**Comment by teodorkasap** [comment]
hi all,
sorry it took a while to respond. 
the notebook for the model is here: https://github.com/teodorkasap/sentiment-analysis-1/blob/master/sentiment-analysis-bidirectional-LSTM-attention.ipynb
I have used two data sets combined, the stanford140 and the Twitter U.S. Airline Sentiment Analysis. The data cleaning can be found here: 
https://github.com/teodorkasap/sentiment-analysis-1/blob/master/preliminary-cleaning-consolidating-tweets.ipynb
after cleaning I have doen stemming and lemmatized tweets: 
https://github.com/teodorkasap/sentiment-analysis-1/blob/master/stemming-lemmatization.ipynb
I am not sure if this is what you had asked from me. If anything is unclear please let me know.

best regards
